port/unix/omrsysinfo.c

```diff
@@ -5707,16 +5902,32 @@ getAbsolutePathOfCgroupSubsystemFile(struct OMRPortLibrary *portLibrary, uint64_
 		goto _end;
 	}
 
-	/* absolute path of the file to be read is: /sys/fs/cgroup/subsystemNames[subsystem]/cgroup/filenName */
-	fullPathLen = portLibrary->str_printf(portLibrary, NULL, (uint32_t)-1, "%s/%s/%s/%s", OMR_CGROUP_MOUNT_POINT, subsystemNames[subsystem], cgroup, fileName);
-	if (fullPathLen > *bufferLength) {
-		*bufferLength = fullPathLen;
-		rc = portLibrary->error_set_last_error_with_message_format(portLibrary, OMRPORT_ERROR_STRING_BUFFER_TOO_SMALL, "buffer size should be %d bytes", fullPathLen);
+	if (OMR_ARE_ALL_BITS_SET(PPG_sysinfoControlFlags, OMR_CGROUP_V1_AVAILABLE)) {
```

Comment:
`OMR_ARE_ALL_BITS_SET` may be slightly more expensive than `OMR_ARE_ANY_BITS_SET` when testing for a single bit: the latter should be used instead.